util/instance.cc

```diff
@@ -154,7 +154,7 @@ bool Instance::init_ssl()
     _last_error= message.str();
     return false;
   }
-#endif // defined(HAVE_CYASSL) && HAVE_CYASSL
+#endif // defined(HAVE_WOLFSSL) && HAVE_WOLFSSL
```

Comment:
If you look at the code, I think this `#endif` actually belongs to the `#if defined(HAVE_SSL) && HAVE_SSL` on line 124. If you agree, please change `WOLFSSL` to `SSL`.